0007-managing-of-customer-apps/0007-managing-of-customer-apps.md

```diff
@@ -0,0 +1,7 @@
+# Managing of customer apps
+
+As a user I’d like to hand over the pager of my apps to Giant Swarm. 
+
+Benefits: We are almost doing it already. The platform gives us quite a lot of tooling to keep the apps of our customers running
+
+Challenge: We need to define clearly what happens once we identify serious problems with an app. How do we give back the pager? How do we make sure we don’t get paged for the same thing again?
```

Comment:
might also need to further extend our monitoring setup in terms of scalability and functionality